bash_completion

```diff
@@ -1702,7 +1702,7 @@ _comp_compgen_available_interfaces()
         else
             ifconfig -a || ip -c=never link show || ip link show
         fi
-    } 2>/dev/null | awk \
+    } 2>/dev/null | command awk \
```

Comment:
In https://github.com/scop/bash-completion/pull/1069#discussion_r1406662760 I tried to say that I'd prefer use of `_comp_awk` throughout (not `command awk`). Now that we have the function, I think it makes perfect sense to make use of it everywhere. Unless of course there's a reason we should not do that, but I'm not aware of any.

Reply:
I used `_comp_awk` in the alternative commit 8372459deba23b86b6eb687665df90c11bda38d9.

Now we do not need Solaris-awk-specific adjustments, so I dropped a1c5c30dc72621d3b36329154e3d1a9babef08c9. Instead, I added commit 9ec2e9eeb0d7af3bd65ba402fb58d53488690146 for the adjustment in the other way.